packages/oasis-actions/src/strategies/aave/open/open.ts

```diff
@@ -124,26 +151,59 @@ async function simulatePositionTransition(
    * It turned out that after opening and then closing a position there might be artifacts
    * Left in a position that make it difficult to re-open it
    */
-  const currentPosition = await dependencies.protocol.getCurrentPosition(
-    {
-      collateralToken: args.collateralToken,
-      debtToken: args.debtToken,
-      proxy: dependencies.proxy,
+  let currentPosition: Position | undefined
+  let protocolData: Unbox<AaveProtocolData> | undefined
+  if (
```

Comment:
As before, the functions look the same so you don't need to check the protocol version.